examples/MHD/input_otv_test.i3d

```diff
@@ -93,6 +93,27 @@ nstat = 1             ! Size arrays for statistic collection
 
 /End
 
+!====================
+&PartiParam
```

Comment:
The `PartiParam` section should be moved to a dedicated PR

Reply:
Fixed this. It was a refuse from the particle development